commit 07902208a94884258a2ddbd09ccfdc41af8d4cfa
Author: Matthieu Herrb <matthieu.herrb@laas.fr>
Date:   Tue May 24 23:34:38 2011 +0200

    fix for an LP64 bug in posterFind()
        
	    posterIoctl(FIO_GETSIZE) expects a pointer to size_t, not int.
	        It makes a difference on LP64 architectures.

--- src/posterLib/posterLib.c.orig	2011-01-24 10:33:09.000000000 +0100
+++ src/posterLib/posterLib.c	2011-05-24 23:20:59.919265000 +0200
@@ -142,7 +142,7 @@
 {
     POSTER_STR *p, *old_p;
     POSTER_ID id;
-    unsigned int size;
+    size_t size;
 
     POSTER_INIT;
 
